### ISSUE TITLE
kde-plasma/kwin-5.22.4-r1 fails to link with gcc-11.2.0

### ISSUE THREAD

**Issue by maffe03** [InBetweenNames/gentooLTO#775]
Just tried to emerge the latest kwin and got a segmentation fault during linking:
`lto1: internal compiler error: Segmentation fault`
I am using gcc-11.2.0 and the issue persists with -j1
Logs:
[build environment](https://github.com/InBetweenNames/gentooLTO/files/6931930/build-environment.txt)
[emerge --info](https://github.com/InBetweenNames/gentooLTO/files/6931934/build-info.txt)
[build log](https://github.com/InBetweenNames/gentooLTO/files/6931911/build-log.txt)

**Comment by zeule** [comment]
Had the same problem, then tried to compile kwin with GCC 10, and it showed a meaningful error:
```
/usr/lib64/libQt5Core.so.5.15.2: undefined reference to `std::__throw_bad_array_new_length()@GLIBCXX_3.4.29'
/usr/lib64/libQt5Core.so.5.15.2: undefined reference to `std::__exception_ptr::exception_ptr::_M_release()@CXXABI_1.3.13'
/usr/lib64/libQt5Core.so.5.15.2: undefined reference to `std::__exception_ptr::exception_ptr::_M_addref()@CXXABI_1.3.13
```
So I ran `lto-rebuild -r` and that solved it (for GCC 11.2).

**Comment by maffe03** [comment]
Yep, worked for me too. Thanks!
